src/hotspot/cpu/ppc/macroAssembler_ppc.hpp

```diff
@@ -359,12 +359,16 @@ class MacroAssembler: public Assembler {
   address call_c(Register function_entry);
   // For tail calls: only branch, don't link, so callee returns to caller of this function.
   address call_c_and_return_to_caller(Register function_entry);
-  address call_c(address function_entry, relocInfo::relocType rt);
+  address call_c(address function_entry, relocInfo::relocType rt = relocInfo::none);
 #else
   // Call a C function via a function descriptor and use full C
   // calling conventions. Updates and returns _last_calls_return_pc.
   address call_c(Register function_descriptor);
-  // For tail calls: only branch, don't link, so callee returns to caller of this function.
+  address call_c(const FunctionDescriptor* function_descriptor, relocInfo::relocType rt);
+  address call_c(address function_entry, relocInfo::relocType rt = relocInfo::none) {
+    return call_c((const FunctionDescriptor*)function_entry, rt);
+  }
+    // For tail calls: only branch, don't link, so callee returns to caller of this function.
```

Comment:
This looks messed up! I requested to move it down, not do duplicate any function declaration. Also, the comment indentation is bad.